src/_locales/cs.json

```diff
@@ -0,0 +1,114 @@
+  "label-quit": "Ukončit",
+  "label-show-lyrics": "Zobrazit text skladby (Beta)",
+  "label-unknown-artist": "Neznámý umělec",
+  "label-unknown-song": "Neznámá skladba",
+  "label-version": "Verze",
+
+  "lastfm-login-authorized": "Přihlášen",
+  "lastfm-login-authorizing": "Probíhá ověření",
+  "lastfm-login-button-text": "Přihlásit se do služby Last.FM",
+  "lastfm-login-error": "Vypadá to, že jste opustili přihlašovací proces. Pokud se tak nestalo, prosím, opakujte přihlášení",
+  "lastfm-login-not-authorized": "Nepřihlášen",
+  "lastfm-logout-button-text": "Odhlásit se ze služby Last.FM",
+
+  "lyrics-failed-message": "Omlouváme se, ale nenašly se žádné texty pro tuto skladbu",
+  "lyrics-loading-message": "Načítání textu skladby...",
+  "lyrics-no-song-message": "Právě nehraje žádná skladba",
+
+  "message-pausing-after-song": "Pozastavení po této skladbě.",
+  "message-pausing-after-song-button": "Nepozastavovat po této skladbě",
+  "message-uncaught-error": "Nastala nespecifikovaná chyba v programu (GPMDP).",
+  "message-uncaught-error-button": "Klikněte zde pro nahlášení tohoto problému na GitHub",
+
+  "modal-confirmTray-title": "Jen abyste věděli",
+  "modal-confirmTray-content": "Zavřením hlavního okna programu (to oč jste se právě pokusili) neukončí program, ale pouze jej minimalizuje do systémové lišty. Pokud chcete tuto možnost změnit, můžete tak učinit v okně \"Nastavení Desktop Přehrávače\".",
+  "modal-confirmOpenPort-content": "Vypadá to, že Váš firewall blokuje službu PlaybackAPI. Můžeme přidat výjimku za Vás, ale upozorňujeme, že to může vyžadovat administrátorské oprávnění",
+  "modal-confirmUpdate-title": "Aktualizace je k dispozici",
+  "modal-confirmUpdate-content": "Hurá, vypadá to, že je zde nová aktualizace pro Google Play Music Desktop Player. Tato aktualizace dodá nálož nových funkcí, opravu chyb a další super věci. Doporučujeme aktualizovat co nejdříve to bude možné, abyste si mohli užít skvělé nové funkce a bezchybné prostředí. <br />Doufáme, že se Vám bude nová aktualizace líbit!",
+  "modal-confirmUninstall-title": "Takže takhle...",
+  "modal-confirmUninstall-content": "Právě jsme zjistili, že máte nainstalovanou starou verzi GPMDP. Pokud jste právě aplikaci aktualizovali, je toto oznámení běžné. Nyní se pokusíme odebrat starou verzi. To \"může\" vyžadovat oprávnění správce. Prosíme Vás o jejich potvrzení, ať můžeme uklidit ten nepořádek.",
+  "modal-goToURL-title": "Otevřete odkaz na Hudba Google Play, aby jste se dostali na:",
+  "modal-welcome-title": "Vítejte ve verzi",
+  "modal-welcome-content": "Doufáme, že se Vám nová verze líbí!",
+
+  "playback-label-next-track": "Další skladba",
+  "playback-label-play-pause": "Přehrát / Pozastavit",
+  "playback-label-previous-track": "Předchozí skladba",
+  "playback-label-stop": "Zastavit",
+  "playback-label-thumbs-down": "Palec dolů",
+  "playback-label-thumbs-up": "Palec nahoru",
+  "playback-label-volume-down": "Zeslabit hlasitost",

```

Comment:
Even the word "zeslabit" is slightly specific in Czech. So I would agree with your first suggestion. I left your original suggestion before but I admit I hesitated as well.